src/modules.c

```diff
@@ -598,7 +596,7 @@ Function global_table[] = {
   (Function) 0,
 #endif
   /* 304 - 307 */
-  (Function) strncpyz,
+  (Function) 0,                   /* was strncpyz() -- use strlcpy() instead */
```

Comment:
Is this necessary? Keeping it in there would allow loading modules compiled for an older version of eggdrop. Removing it makes them crash when they use strncpyz. The rest of the changes is fine.

Reply:
We did this also with functions like egg_strcasecmp().

Reply:
maybe it was a mistake not to do this also for my_memcpy() and friends.
so lets do it proper from now on.
done, ready for review again.